uro98/assignment1/Anagram.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class Anagram {
```

Comment:
Could you please add javadoc comments to public methods? API should have some documentation that would make it easy for people not familiar with your class to use it in their code.